packages/react/src/Feedback.tsx

```diff
@@ -0,0 +1,41 @@
+import React, {
+  useState,
+  type ReactElement,
+  type ComponentPropsWithoutRef,
+} from 'react';
+
+import { useMarkpromptContext } from './context.js';
+import { ThumbsDownIcon, ThumbsUpIcon } from './icons.js';
+
+export function Feedback(
+  props: ComponentPropsWithoutRef<'aside'>,
+): ReactElement {
+  const { submitFeedback } = useMarkpromptContext();
+  const [feedback, setFeedback] = useState<boolean>();
+
+  async function handleFeedback(helpful: boolean): Promise<void> {
+    await submitFeedback(helpful);
+    setFeedback(helpful);
+  }
+
+  return (
+    <aside {...props}>
+      <h3>Was this response helpful?</h3>
+
+      {typeof feedback !== 'boolean' && (
+        <div>
+          <button aria-label="Yes" onClick={() => handleFeedback(true)}>
+            <ThumbsUpIcon width={16} height={16} />
```

Comment:
Should be able to use a label "Yes/No" instead of icons. Ideally, icons should also be configurable (without being limited only to React applications with headless). Not a priority now though.